src/data-source/collection-browser-data-source.ts

```diff
@@ -1220,7 +1221,14 @@ export class CollectionBrowserDataSource
     const tiles: TileModel[] = [];
     results?.forEach(result => {
       if (!result.identifier) return;
-      tiles.push(new TileModel(result));
+      const requestSource: HitRequestSource = this.host.baseQuery
+        ? 'search_query'
+        : this.host.withinProfile
+          ? 'profile_tab'
+          : this.host.withinCollection
+            ? 'collection_members'
+            : 'unknown';
```

Comment:
Suggestion: pull this ternary out to a getter to help improve the readability of this `forEach()` (and maybe make it a series of `if`s instead of a ternary, also for readability). Also, since it's a one-time computed value, you can move it outside of the `forEach` so it doesn't have to compute it each time. ie:
```
private get hitRequestSource(): HitRequestSource {
  if (this.host.baseQuery) return 'search_query';
  if (this.host.withinProfile) return 'profile_tab';
  ...
}
...
const requestSource = this.hitRequestSource;
forEach( ...
  new TileModel(result, requestSource)
)
```

Reply:
Done, thanks for catching that!